File: backend-expressjs/src/routes/transfer.js
import express from "express";
import {
  transferMoney,
  getTransactions,
} from "../controllers/transferController.js";
import authMiddleware from "../middleware/authMiddleware.js";

const router = express.Router();

router.post("/transfer", authMiddleware, transferMoney);
router.get("/transactions", authMiddleware, getTransactions);

export default router;

File: backend-rust/src/routes/transfer.rs
use actix_web::web;
use crate::controllers;

pub fn transfer_routes(cfg: &mut web::ServiceConfig) {
    cfg.service(
        web::scope("/api")
            .route("/transfer", web::post().to(controllers::transfer_money))
            .route("/transactions", web::get().to(controllers::get_transactions))
    );
}
